docs/recording_provenance.md

```diff
@@ -115,6 +115,33 @@ mutation {
 }
 ```
 
+### Representing Mutability
+
+Working through our [example domain definitions](https://examples.btp.works/),
+one may notice that some define attributes like "location" that may seem as if
+they could be mutable. They are *not* mutable: the above rules on
+contradiction hold firm. Such examples use entities or agents in fixed
+locations, and illustrate other facets of modeling domains in Chronicle.
```

Comment:
Possibly say something explicit about having to model changes to attributes as provenance?